docs/en/operations/server_settings/settings.md

```diff
@@ -558,9 +560,8 @@ Use the following parameters to configure logging:
 
 The path to the directory containing data.
 
-!!! warning "Attention"
-    The trailing slash is mandatory.
-
+!!! Note:
+The end slash is mandatory.
```

Comment:
better to leave it as before. "trailing slash" is kinda idiom in IT.

See for example http://sites.middlebury.edu/librarynews/2017/05/12/the-story-of-the-trailing-slash/

Reply:
Yep, there are some more issues like this, it was written by not-that-technical translator.